indymeet/templates/puput/base.html

```diff
@@ -28,12 +28,12 @@ <h2 class="pl-5">{{ blog_page.description }}</h2>
 {% endblock blog_header %}
 {% wagtailuserbar %}
 <div class="container container-blog mx-auto">
-    <div class="flex">
-        <div class="basis-3/4">
+    <div class="grid grid-cols-1 md:grid-cols-4">
+        <div class="col-span-1 md:col-span-3">
             {% block extra_content %}{% endblock extra_content %}
         </div>
 
-        <div class="basis-1/4">
+        <div class="col-span-1 md:col-span-1">
```

Comment:
```suggestion
        <div class="col-span-1">
```
Or did you want it to change at md?

Reply:
I had both to make it easier to change in the future. I was real close to slimming it down. If you think it's not worth it I'll change it.

Reply:
Nah that's fine 👍